docs/querying/sql-functions.md

```diff
@@ -1077,11 +1077,42 @@ Calculates the base-10 of the numeric expression.
 
 ## LOOKUP
 
-`LOOKUP(<CHARACTER>, <CHARACTER>[, <CHARACTER>])`
+Look up `expr` in a registered [query-time lookup table](lookups.md) named `lookupName`. Returns `replaceMissingValueWith`, which defaults to `null`, when the `expr` is null or when the lookup does not contain a value for `expr`.
 
-**Function type:** [Scalar, string](sql-scalar.md#string-functions)
+* **Syntax:** `LOOKUP(expr, lookupName[, replaceMissingValueWith])`
+* **Function type:** Scalar, string
+
+<details><summary>Example</summary>
+
+The following example uses a map lookup table, named `acronym_to_name` made with the following key-value pairs:
+
+```json
+{
+	“SJU”: Luis Munoz Marin International Airport”,
+	“IAD”: “Dulles International AirportDulles International Airport”
```

Comment:
Can you update this to
* Change tabs to spaces
* Used non-stylized quotation marks
* Add missing quotation marks for the first line

Reply:
- [x] Change tabs to spaces
- [x] Used non-stylized quotation marks
- [x] Add missing quotation marks for the first line